renderer/main.js

```diff
@@ -314,6 +314,8 @@ function dispatch (action, ...args) {
     saveStateThrottled()
   }
   if (action === 'saveState') {
+    state.saved.window = state.saved.window || {}
+    state.saved.window.bounds = args[0]
```

Comment:
I agree with your suggestion, did not notice that _window_ was a available in the renderer. Dispatching bounds is more cumbersome.